### ISSUE TITLE
Cannot install on Laravel 5.7.x even though docs say it's supported

### ISSUE THREAD

**Issue by VetSoftKeith** [thedevdojo/voyager#3564]
## Version information

 - Laravel: v5.7.4
 - Voyager: v1.1
 - PHP: 7.2.9
 - Database: MariaDB 10.3.9

## Description

Unable to install on Laravel 5.7.x.  Voyager's Composer dependencies are not updated to allow installation on Laravel 5.7

## Steps To Reproduce
1. Install Laravel 5.7.x per instruction in Laravel docs
2. `composer require tcg/voyager`
3. Composer goes bananas with messages about incompatible Laravel/Illuminate/Doctrine versions
4. See error here: https://hastebin.com/gazofuwaya.sql

## Expected behavior
I expected Voyager to be compatible out of the box with Laravel 5.7 since the docs say it is compatible

## Screenshots
Error message too long for a single screenshot.  Copy/pasta of the errors at: https://hastebin.com/gazofuwaya.sql

## Additional context
Even attempting a hard version constraint of 1.1.5 of Voyager still errors out saying that the doctrine/dbal constraint of 2.8.0 that Laravel 5.7 uses is too much.  I'm aware of #3519, but it was closed stating that 5.7 is now supported

**Comment by homer3** [comment]
@keesh7663 thanks for submitting the issue.

I just tested on my end, and seems to be working fine. I've also has @marktopper try it out and it seems to be working fine.

Can you remove your `composer.lock` and `vendor` folder. Then paste this into your `composer.json`

```
{
    "name": "laravel/laravel",
    "description": "The Laravel Framework.",
    "keywords": ["framework", "laravel"],
    "license": "MIT",
    "type": "project",
    "require": {
        "php": "^7.1.3",
        "fideloper/proxy": "^4.0",
        "laravel/framework": "5.7.*",
        "laravel/tinker": "^1.0",
        "tcg/voyager": "^1.1"
    },
    "require-dev": {
        "beyondcode/laravel-dump-server": "^1.0",
        "filp/whoops": "^2.0",
        "fzaninotto/faker": "^1.4",
        "mockery/mockery": "^1.0",
        "nunomaduro/collision": "^2.0",
        "phpunit/phpunit": "^7.0"
    },
    "autoload": {
        "classmap": [
            "database/seeds",
            "database/factories"
        ],
        "psr-4": {
            "App\\": "app/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "extra": {
        "laravel": {
            "dont-discover": [
            ]
        }
    },
    "scripts": {
        "post-root-package-install": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "@php artisan key:generate"
        ],
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover"
        ]
    },
    "config": {
        "preferred-install": "dist",
        "sort-packages": true,
        "optimize-autoloader": true
    },
    "minimum-stability": "dev",
    "prefer-stable": true
}
```

Then, run `composer install` again :)

Let me know if that helps.

**Comment by VetSoftKeith** [comment]
So in my dev environment for my site, I reverted my `composer.json` to what you listed above, ran a `composer update`, then put it back to where I had it and ran `composer update` again and it seemed to resolve the issue.  I dug through the `composer.json` for each package I've installed and wasn't able to find a constraint for `doctrine/dbal` higher `^2.6` so not entirely certain how I ended up with 2.8.  Thanks for the assist.
